### PR TITLE
Write about getting hired

### PR DESCRIPTION
**Pull request by jgorset**
I thought I'd just open source this. Because that's a cool thing to do.

### CODE REVIEW

File: GETTING_HIRED.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Getting hired
```
Comment (NSElvis) [inline GETTING_HIRED.md]: Should this be "Oh, you where hired!", "Getting hired" implies steps on how to get hired.

Comment (jgorset) [inline GETTING_HIRED.md]: Good point, but what about the file name?

Comment (NSElvis) [inline GETTING_HIRED.md]: We should change that too, what about `YOU_GOT_HIRED`

**Comment by NSElvis** [comment]
@timkurvers @jgorset: I don't necessarily agree but 2 vs 1 :) majority wins.

**Comment by jgorset** [comment]
Ha ha, "The Return Of The Newlines". Either way you get a cookie for contributing! Thanks!

File: GETTING_HIRED.md (hunk 1 above)
Comment (jgorset) [inline GETTING_HIRED.md]: Agreed! Done!

**Comment by NSElvis** [comment]
:tada: :tada: :tada:

**Comment by zenangst** [comment]
@jgorset :cake:

**Comment by jgorset** [comment]
Thanks guys! There's a bunch more we need to write about like getting a working permit and all that, but we'll get around to it.

**Comment by sindrenm** [comment]
http://baymard.com/blog/line-length-readability

**Comment by jgorset** [comment]
So code should be 90 characters, but markdown should be 60?

**Comment by timkurvers** [comment]
Nice find @sindrenm, that's a thingie I was looking for!

**Comment by zenangst** [comment]
@jgorset I think the code values depend on which language you write. In Objective-C, 90 characters would be one method name (without arguments) :grin:

**Comment by sindrenm** [comment]
Well, optimal readability is achieved when the character length is at around 60. However, there are differences in reading monospace characters and err … “normal” characters, probably the most obvious one being that punctuation takes up the exact same space as any other character in monospace fonts. That impacts the length of the line, and you probably read Markdown code in monospace, but the parsed result in a proportional font. :stuck_out_tongue:

**Comment by NSElvis** [comment]
@jgorset @zenangst Optimal length on Objective-C methods is 93432432 characters.

**Comment by sindrenm** [comment]
:grinning:

**Comment by zenangst** [comment]
@NSElvis what yourself, you might go out of bounds!

**Pull request merged**